collect_app/src/main/java/org/odk/collect/android/widgets/QuestionWidget.java

```diff
@@ -55,6 +57,7 @@
         extends RelativeLayout
         implements Widget, AudioPlayListener {
 
+    protected Bundle state;
```

Comment:
I would make this private and limit access via methods to avoid `state` being accidentally set null in a child class.